### PR TITLE
Use schema name as suffix

### PR DESCRIPTION
**Pull request by peppy**
- [x] Depends on https://github.com/ppy/osu-elastic-indexer/pull/151

This simplifies the naming of things related to ES indexing, by naming the indices after the schema versions.

```bash

root@path-to-ranking:~/repos/osu-elastic-indexer/osu.ElasticIndexer# SCHEMA=wang dotnet run queue watch
WARNING: No current schema set, will set new schema as current
Running queue with schema version scores_wang

Creating new index `scores_wang`.
Schema switched to current: scores_wang
Updating `scores` alias to `scores_wang`...
Starting queue processing (Backlog of 0)..
stats: queue:0 inflight:0 dequeued:0 processed:0 errors:0
stats: queue:0 inflight:0 dequeued:0 processed:0 errors:0
^CShutting down..
Bye!
stats: queue:0 inflight:0 dequeued:0 processed:0 errors:0
root@path-to-ranking:~/repos/osu-elastic-indexer/osu.ElasticIndexer# dotnet run index list
# Redis tracking

Active schemas: scores_wang
Current schema: scores_wang

# Elasticsearch indices (1)

scores_wang (454b)
- aliases: scores
- status: open
- docs: 0 (0 deleted)


root@path-to-ranking:~/repos/osu-elastic-indexer/osu.ElasticIndexer# SCHEMA=wang-newer dotnet run queue watch
WARNING: Starting processing for schema version scores_wang-newer which is not current (current schema is scores_wang)
Running queue with schema version scores_wang-newer

Creating new index `scores_wang-newer`.
Starting queue processing (Backlog of 0)..
stats: queue:0 inflight:0 dequeued:0 processed:0 errors:0
^CShutting down..
Bye!

root@path-to-ranking:~/repos/osu-elastic-indexer/osu.ElasticIndexer# dotnet run index alias wang-newer
Updating `scores` alias to `scores_wang-newer`...

root@path-to-ranking:~/repos/osu-elastic-indexer/osu.ElasticIndexer# dotnet run index list
# Redis tracking

Active schemas: scores_wang-newer,scores_wang
Current schema: scores_wang-newer

# Elasticsearch indices (2)

scores_wang (454b)
- aliases:
- status: open
- docs: 0 (0 deleted)

scores_wang-newer (454b)
- aliases: scores
- status: open
- docs: 0 (0 deleted)
```

### CODE REVIEW

**Review by notbakaneko — CHANGES_REQUESTED** [review]
https://github.com/ppy/osu-elastic-indexer/blob/fbd0c48346ed053dfcef8221f3195176268696c5/osu.ElasticIndexer/IndexQueueProcessor.cs#L15
The suffix needs to match the one stored in `schema/active-schemas`; it's like that so other programs can just get the value from redis and use it directly to push to `score-index-${valueFromRedis}`

---
https://github.com/ppy/osu-elastic-indexer/blob/fbd0c48346ed053dfcef8221f3195176268696c5/osu.ElasticIndexer/OsuElasticClient.cs#L16
I think this can be static?

**Comment by peppy** [comment]
Good catch on the queue name, I also noticed this today during testing with score processing. Have applied the two pieces of feedback 👍 .

**Comment by notbakaneko** [comment]
I was thinking the key for the queue itself should have some kind of prefix so there's no chance of it colliding with the `score-index:active-schema` and `score-index:schema` keys, but, it looks like `:` can't be used for index names, so that won't be a problem anyway.

File: osu.ElasticIndexer/AppSettings.cs [hunk 1]
```diff
@@ -37,5 +40,7 @@ static AppSettings()
         public static string RedisHost { get; }
 
         public static string Schema { get; }
+
+        public static string AliasName { get; } = $"{AppSettings.Prefix}scores";
```
Comment (notbakaneko) [inline osu.ElasticIndexer/AppSettings.cs:44]: This initializer runs before `Prefix` gets set if I'm not mistaken 👀

**Review by notbakaneko — APPROVED** [review]
looks ok so far

**Pull request merged**